### PR TITLE
Use pipenv for virtualenv

### PR DESCRIPTION
**Pull request by sidharthv96**

### CODE REVIEW

**Comment by joicemjoseph** [comment]
http://www.bogotobogo.com/python/python_virtualenv_virtualenvwrapper.php

virtualenvwrapper does the same. it is easy to work no need for activation and deactivation. all of them are automatically managed.

**Comment by sadu** [comment]
dupilcates #23 and #26

File: README.md [hunk 1]
```diff
@@ -55,23 +55,26 @@ cp .env.example .env
...
+pipenv --python 3.7
+pipenv install
+pipenv shell
 ```
 
 Run Database migrations
 
 ```
-python3 manage.py migrate
+python manage.py migrate
```
Comment (kira0204) [inline README.md]: This command would fail in the systems where default python is python2 because of the syntax in `manage.py`

**Comment by salmanulfarzy** [comment]
@sadu Issues with having multiple dependency management solutions :v:

**Comment by sadu** [comment]
@salmanulfarzy haha yes figured out that now, anyways we can have only one merged I guess

File: README.md (hunk 1 above)
Comment (sidharthv96) [reply README.md]: pipenv shell changes python to default to python3.

**Comment by jilvin** [comment]
@sidharthv96 Duplicates #23. Please review #23 and contribute to the same. Else clarify.

**Comment by remedcu** [comment]
You may close this PR.

Related ones: #23 #26 #33 #392 

I think just #392 (if required) is enough.

@syam3526 @vigneshhari @biswaz @vrthra

**Comment by bobinson** [comment]
Thank you. @remedcu

**Pull request closed without merging**